npm start throws error b/c Flask(?)

run `npm start` on master

```
Traceback (most recent call last):
  File "run.py", line 43, in <module>
    raise ImportError('{0}\n\n{1}'.format(e, IMPORT_ERROR))
ImportError: cannot import name FlaskForm


==============================================
It seems like you don't have all dependencies.
Please re-run:
    pip install -r requirements.txt
==============================================
```

`pip install -r config/requirements.txt `

```
Requirement already satisfied: Babel==2.3.4 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 1))
Requirement already satisfied: blinker==1.4 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 2))
Requirement already satisfied: Flask==0.12 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 3))
Requirement already satisfied: Flask-Limiter==0.9.3 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 4))
Requirement already satisfied: Flask-SQLAlchemy==2.1 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 5))
Requirement already satisfied: Flask-WTF==0.14.2 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 6))
Requirement already satisfied: GitHub-Flask==3.1.5 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 7))
Requirement already satisfied: nose==1.3.7 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 8))
Requirement already satisfied: pep8==1.7.0 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 9))
Requirement already satisfied: Pillow==4.0.0 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 10))
Requirement already satisfied: requests==2.13.0 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 11))
Requirement already satisfied: ua-parser==0.7.2 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 12))
Requirement already satisfied: WTForms==2.1 in /Library/Python/2.7/site-packages (from -r config/requirements.txt (line 13))
Requirement already satisfied: pytz>=0a in /System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python (from Babel==2.3.4->-r config/requirements.txt (line 1))
Requirement already satisfied: itsdangerous>=0.21 in /Library/Python/2.7/site-packages (from Flask==0.12->-r config/requirements.txt (line 3))
Requirement already satisfied: Werkzeug>=0.7 in /Library/Python/2.7/site-packages (from Flask==0.12->-r config/requirements.txt (line 3))
Requirement already satisfied: Jinja2>=2.4 in /Library/Python/2.7/site-packages (from Flask==0.12->-r config/requirements.txt (line 3))
Requirement already satisfied: click>=2.0 in /Library/Python/2.7/site-packages (from Flask==0.12->-r config/requirements.txt (line 3))
Requirement already satisfied: six>=1.4.1 in /System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python (from Flask-Limiter==0.9.3->-r config/requirements.txt (line 4))
Requirement already satisfied: limits in /Library/Python/2.7/site-packages (from Flask-Limiter==0.9.3->-r config/requirements.txt (line 4))
Requirement already satisfied: SQLAlchemy>=0.7 in /Library/Python/2.7/site-packages (from Flask-SQLAlchemy==2.1->-r config/requirements.txt (line 5))
Requirement already satisfied: olefile in /Library/Python/2.7/site-packages (from Pillow==4.0.0->-r config/requirements.txt (line 10))
Requirement already satisfied: MarkupSafe>=0.23 in /Library/Python/2.7/site-packages (from Jinja2>=2.4->Flask==0.12->-r config/requirements.txt (line 3))
```
@karlcow maybe you could help?

Might be related to https://github.com/webcompat/webcompat.com/pull/1307 ?

(Site note...  `pip install -r requirements.txt` -> `pip install -r config/requirements.txt `)

for what is worth `npm run start` fails for me too.
but it's not how I start the server.

I start with `python run.py` 

```
→ npm run start

> webcompat@ start /Users/karl/code/webcompat.com
> source env/bin/activate && python run.py

('secrets', '/Users/karl/code/webcompat.com')
Traceback (most recent call last):
  File "run.py", line 43, in <module>
    raise ImportError('{0}\n\n{1}'.format(e, IMPORT_ERROR))
ImportError: No module named flask_wtf


==============================================
It seems like you don't have all dependencies.
Please re-run:
    pip install -r requirements.txt
==============================================


npm ERR! Darwin 16.4.0
npm ERR! argv "/Users/karl/.nvm/versions/node/v4.0.0/bin/node" "/Users/karl/.nvm/versions/node/v4.0.0/bin/npm" "run" "start"
npm ERR! node v4.0.0
npm ERR! npm  v2.14.2
npm ERR! code ELIFECYCLE
npm ERR! webcompat@ start: `source env/bin/activate && python run.py`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the webcompat@ start script 'source env/bin/activate && python run.py'.
npm ERR! This is most likely a problem with the webcompat package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     source env/bin/activate && python run.py
npm ERR! You can get their info via:
npm ERR!     npm owner ls webcompat
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/karl/code/webcompat.com/npm-debug.log
```

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -247,7 +247,7 @@ cd webcompat.com
 # check out submodules
 npm run module
 # initializing project
-[sudo] npm run init
+[sudo] npm run setup
 ```
 
 **Note**: if you got an error message, you may need to [install pip](#installing-pip) before running `make install` again.

--- a/package.json
+++ b/package.json
@@ -40,12 +40,12 @@
     "suitcss-utils-display": "^0.4.0"
   },
   "scripts": {
-    "init": "npm run module && npm run virtualenv && npm run pip && npm install && npm run config",
+    "setup": "npm run module && npm run virtualenv && npm install && npm run config",
     "watch" : "grunt watch",
     "build": "grunt",
     "module": "git submodule init && git submodule update",
     "start": "source env/bin/activate && python run.py",
-    "virtualenv" : "pip install virtualenv && virtualenv env &&  source env/bin/activate",
+    "virtualenv" : "pip install virtualenv && virtualenv env && source env/bin/activate && npm run pip",
     "pip": "pip install -r config/requirements.txt",
     "config": "cp config/secrets.py.example config/secrets.py",
     "project-update": "pip install --upgrade pip && npm run backend-install && npm update"

